etc/kayobe/inventory/group_vars/all/selinux

```diff
@@ -0,0 +1,9 @@
+---
+# Configure SELinux in permissive mode when configuring a Rocky Linux 9 host.
+selinux_state: "{{ 'permissive' if ansible_facts.os_family == 'RedHat' and ansible_facts.distribution_major_version == '9' else 'disabled' }}"
```

Comment:
Shorter but maybe less clear? What do you think?
```suggestion
selinux_state: "{{ 'disabled' if disable_selinux_do_reboot else 'permissive' }}"
```

Reply:
Thanks but I am not keen on this suggestion, as you have to understand the logic of the second variable to know what the first variable evaluates to.

Reply:
Thanks but I am not keen on this suggestion, as you have to understand the logic of the second variable to know what the first variable evaluates to.